### PR TITLE
PPF-367-404 page

### PR DESCRIPTION
**Pull request by vhande**
### Added
-404 page

Screenshot

---
Ticket: https://jira.uitdatabank.be/browse/PPF-367

### CODE REVIEW

File: app/Exceptions/Handler.php [hunk 1]
```diff
@@ -62,14 +62,14 @@ public function register(): void
...
+            if ($exception instanceof HttpException && $exception->getStatusCode() >= 500) {
                 return $this->handle500Error($exception);
             }
         });
     }
 
     protected function handle500Error(Exception $exception): Response
     {
-        return Inertia::render('Error');
+        return Inertia::render('Error', ['statusCode' => '500']);
```
Comment (simon-debruijn) [inline app/Exceptions/Handler.php:73]:
```suggestion
        return Inertia::render('Error', ['statusCode' => 500]);
```

the status code can be a number

File: resources/ts/Pages/Error.tsx [hunk 2]
```diff
@@ -3,37 +3,57 @@ import React from "react";
...
+import { ButtonSecondary } from "../../ts/Components/ButtonSecondary";
+import { Link } from "../../ts/Components/Link";
+import { ErrorImage } from "../Components/ErrorImage";
+import { Heading } from "../Components/Heading";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faChevronRight } from "@fortawesome/free-solid-svg-icons";
 
-const Error = () => {
+type Props = {
+  statusCode: string;
```
Comment (simon-debruijn) [inline resources/ts/Pages/Error.tsx:14]:
```suggestion
  statusCode: number;
```

File: resources/ts/Pages/Error.tsx [hunk 3]
```diff
@@ -3,37 +3,57 @@ import React from "react";
...
+};
+
+const Error = ({ statusCode }: Props) => {
   const { t } = useTranslation();
   const navigateBack = () => window.history.back();
 
+  if (statusCode === "500")
```
Comment (simon-debruijn) [inline resources/ts/Pages/Error.tsx:21]:
```suggestion
  if (statusCode === 500)
```

Comment (simon-debruijn) [reply resources/ts/Pages/Error.tsx:21]: Maybe it is more logical to check for 404 and return the jsx. And the default return at the end is a 500 return

File: resources/ts/Components/ErrorImage.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import Error from "../../images/Error.svg";
```
Comment (simon-debruijn) [inline resources/ts/Components/ErrorImage.tsx:2]:
```suggestion
import NotFoundError from "../../images/NotFoundError.svg";
```

**Review by simon-debruijn — CHANGES_REQUESTED** [review]

**Review by brampauwelyn — APPROVED** [review]
BIUTIFUL ⭐

**Review by simon-debruijn — APPROVED** [review]

**Pull request merged**